solutions/usaco-856.mdx

```diff
@@ -208,5 +208,46 @@ public class TheBucketList2 {
 ```
 	
 </JavaSection>
+<PySection>
+```py
+fin = open("blist.in", "r")
+fout = open("blist.out", "w")
+
+event = [0 for i in range(1001)]  # initialize all elements to 0
+
+max_time = 1000
+n = int(fin.readline().strip())
+
+for i in range(n):
+	start, end, amt = map(int, fin.readline().strip().split())
+
+	# at time start, cow i needs to be milked, so 
+	# we'll need amt additional buckets (hence adding amt)
+	event[start] += amt
+
+	# at time end, cow i is done milking, so we can free up 
+	# amt buckets (hence subtracting amt)
+	event[end] -= amt
 
+# res represents the maximum number of buckets we'll need 
+# to use at any given time and cur represents 
```

Comment:
```py
"""
multiline comments should
be like this
"""
```
also at this point there's more comments than code
consider just putting an editorial or something

Reply:
lmao check the c++ solution

Reply:
then fix the c++ solution as well

Reply:
it's not needed, and the comments fit well anyways